packages/mdc-form-field/README.md

```diff
@@ -7,141 +7,103 @@ path: /catalog/input-controls/form-fields/
 
 # Form Fields
 
-MDC Form Field provides an `mdc-form-field` helper class for easily making theme-aware, RTL-aware
-form field + label combos. It also provides an `MDCFormField` class for easily making input ripples
-respond to label events.
+MDC Form Field aligns a form field (for example, a checkbox) with its label and makes it RTL-aware.
+When used with JavaScript, it can activate a [ripple](../mdc-ripple) effect upon interacting with the label.
 
 ## Installation
 
 ```
 npm install @material/form-field
 ```
 
-## CSS Usage
+## Demos
 
-The `mdc-form-field` class can be used as a parent element, with any combination of adjacent `input` and `label` as
-immediate children:
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item--link">
+    <a href="https://material-components-web.appspot.com/checkbox.html">Demo with checkbox</a>
+  </li>
+  <li class="icon-list-item icon-list-item--link">
+    <a href="https://material-components-web.appspot.com/radio.html">Demo with radio button</a>
+  </li>
+</ul>
 
-```html
-<div class="mdc-form-field">
-  <input type="checkbox" id="input">
-  <label for="input">Input Label</label>
-</div>
-```
+## Basic Usage
 
-By default, this will position the label after the input. You can change this behavior using the
-`align-end` modifier class.
+### HTML Structure
+
+The `mdc-form-field` class is used as a parent element, with any combination of adjacent `input` and `label` elements as
+immediate children:
 
 ```html
-<div class="mdc-form-field mdc-form-field--align-end">
+<div class="mdc-form-field">
```

Comment:
I apparently failed to notice this before... Should we just remove this first example and combine the first and next paragraph, jumping straight to the part that talks about using it with MDC Checkbox and Radio?

"MDC Form Field operates as a parent element and works with any type of immediate child element whose successive sibling is a `label` element. This means ..."